tests/test_transforms/test_embeddings/test_models/utils.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+import pandas as pd
+
+
+def cut_nan_timestamps(df: pd.DataFrame) -> np.ndarray:
```

Comment:
I mentioned about additional preparing of df in a case of `full_series` mode.

It is that

Reply:
Why is this only for tests? If you feed data with nans at the end into the model, it won’t be cut off, right?